### PR TITLE
Making footer accessible for screen readers

### LINKED ISSUE #50: Remove Icomoon dependency
For the MVP, we pulled in Icons in a file directly from Icomoon's site and used an NPM library to display the icons. We should remove the Icomoon dependency and render these icons like regular SVGs.

AC:
1) Create svg files for the two icons we're using (the key and the home in the header)
1) Remove icomoon react library
1) Render the icons as svg images

### PR DESCRIPTION
**Pull request by sharonwarner**
Resolves #50 

- Removes icomoon-react library
- Adds icons as SVG files
- Adds alt text for all icons

icons

### CODE REVIEW

File: src/client/components/Footer/index.js [hunk 1]
```diff
@@ -29,43 +27,43 @@ class Footer extends React.PureComponent {
...
-              <IcomoonReact
-                iconSet={iconSet}
-                color="#ffffff"
-                size={20}
-                icon="facebook"
+              <img
+                src="images/facebook.svg"
+                width="15"
+                height="15"
+                alt="facebook icon"
```
Comment (idealisms) [inline src/client/components/Footer/index.js:34]: Should the alt text be localized?

Comment (sharonwarner) [reply src/client/components/Footer/index.js:34]: I got out of practice when we put all the content in for the launch! Thanks for catching this. Updating now!

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -11,5 +11,14 @@
...
     "5": "More Resources"
   },
-  "buttonNextPrefix": "Next: "
+  "buttonNextPrefix": "Next: ",
+  "iconAltText": {
+    "home": "home icon",
+    "key": "key icon",
+    "facebook": "Facebook icon",
+    "twitter": "Twitter icon",
+    "linkedIn": "Linked In icon",
```
Comment (idealisms) [inline public/locales/en/translation.json:20]: LinkedIn (and the same for the Spanish translation)

File: public/locales/en/translation.json [hunk 3]
```diff
@@ -11,5 +11,14 @@
...
+    "instagram": "Instagram icon",
+    "youtube": "Youtube icon"
```
Comment (idealisms) [inline public/locales/en/translation.json:22]: YouTube (and the same for the Spanish translation)

File: src/client/components/Header/index.js [hunk 4]
```diff
@@ -1,73 +1,75 @@
...
+  return (
+    <header className="header border-bottom border-secondary">
+      <Navbar className="justify-content-between" variant="custom" bg="primary">
+        <Navbar.Brand href="https://ca.gov">
+          <img
+            src="images/Ca-Gov-Logo-Gold.svg"
+            width="30"
+            height="30"
+            className="d-inline-block align-top"
+            alt="California gov logo"
```
Comment (idealisms) [inline src/client/components/Header/index.js:18]: Can we translate this too?

**Pull request merged**